src/decode.rs

```diff
@@ -4879,6 +4879,18 @@ pub(crate) fn rav1d_decode_frame_exit(
     if c.fc.len() > 1 && retval.is_err() {
         cf.fill_with(Default::default);
     }
+
+    if !retval.is_err() && c.fc.len() > 1 && c.strict_std_compliance {
+        if f.refp.iter().any(|rf| {
+            rf.p.frame_hdr.is_some()
+                && rf.progress.as_ref().unwrap()[1].load(Ordering::Relaxed) == FRAME_ERROR
+        }) {
+            retval = Err(EINVAL);
+            fc.task_thread.error.store(1, Ordering::Relaxed);
+            f.sr_cur.progress.as_mut().unwrap()[1].store(FRAME_ERROR, Ordering::Relaxed);
```

Comment:
```suggestion
            fc.task_thread.error.store(1, Ordering::SeqCst);
            f.sr_cur.progress.as_mut().unwrap()[1].store(FRAME_ERROR, Ordering::SeqCst);
```

Reply:
I kept `Ordering::Relaxed` for `fc.task_thread.error`. There is an existing `store` for that variable in the same function that uses `Ordering::Relaxed` and I am assuming it's correct.

Reply:
That might be different.  Did C explicitly do a relaxed store there?  I think it's better to do seq cst here and match what C is doing.  Unless you have an idea why specifically relaxed would be okay here, I think it's safer to just match C.

Reply:
You're right. I fixed it to match the C code.